tests/Fixtures/inc/Engine/Cache/Config/ConfigSubscriber/changeCacheRejectUriWithPermalink.php

```diff
@@ -0,0 +1,101 @@
+<?php
```

Comment:
(That's a dev matter so it won't affect the PR itself and not a blocker)
I know that you need to write less code and not to duplicate ourselves but this fixture isn't readable for me, I believe you need to make it simpler by writing the pattern manually for each case below, so it'd be easier for anyone to add cases whenever u want